cmd/helpers.go

```diff
@@ -46,3 +49,34 @@ func UserConfirmYes(ctx *cmd.Context) error {
 	}
 	return nil
 }
+
+// UserConfirmName returns an error if we do not read a "name" of the model/controller/etc from user
+// input.
+func UserConfirmName(verificationName string, objectType string, ctx *cmd.Context) error {
+	fmt.Fprintf(ctx.Stdout, "\nTo continue, enter the name of the %s to be destroyed:", objectType)
```

Comment:
An complete knit pick, but the other function uses a package level private var for the initial print. For symmetry, you should od the same here (The fact you're templating shouldn't make a difference)

Reply:
Done